spec/API_specification/index.rst

```diff
@@ -3,6 +3,18 @@
 API specification
 =================
 
+A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.
+
+-   Positional parameters must be `positional-only <https://www.python.org/dev/peps/pep-0570/>`_ parameters when the function signature indicates this (as denoted by `/`). See :ref:`function-and-method-signatures`.
```

Comment:
```suggestion
-   When a function signature includes a `/`, positional parameters must be `positional-only <https://www.python.org/dev/peps/pep-0570/>`_ parameters. See :ref:`function-and-method-signatures`.
```

Rephrasing to avoid ambiguous antecedent.